content/docs/faq.md

```diff
@@ -90,7 +90,7 @@ We never send Scratch account information outside of your browser. See [the exte
 To enable addons, first go to the settings by:
 
 - Opening the popup and clicking the gear icon, or:
-- by going to [https://scratch.mit.edu/scratch-addons-extension/settings](https://scratch.mit.edu/scratch-addons-extension/settings). Note: this URL only works for Scratch Addons users.
+- by going [here](https://scratch.mit.edu/scratch-addons-extension/settings). Note: this URL only works for Scratch Addons users.
```

Comment:
Avoid "here" links if possible - they're bad for accessibility. In this case using the URL is probably better.

Reply:
@mxmou , is it fine to use a link like this : by going to [the settings page](https://scratch.mit.edu/scratch-addons-extention/settings)

Reply:
That would be weird because these are instructions for accessing the settings page.

Reply:
Also, spotted an issue :|. You CAN'T access the settings page unless you click on the gear. That is because the settings page is localized. I'm fixing that now :).
![image](https://user-images.githubusercontent.com/82447031/181232500-6df4844b-4eb2-460d-8f88-fee99cea04c5.png)

Reply:
Also, isn't the square brackets redundant in this case @mxmou